### PR TITLE
util: genElapsedTime module and test adjustments

### PR DESCRIPTION
**Pull request by habibayassin**

### CODE REVIEW

File: flow/util/test/test_genElapsedTime.py [hunk 1]
```diff
@@ -11,15 +11,17 @@
 from datetime import datetime
```
Comment (vvbandeira) [inline flow/util/test/test_genElapsedTime.py]:
```py
from unittest.mock import MagicMock
import io
from datetime import datetime
import argparse  # argument parsing
from pathlib import Path
```
Let's cleanup these unused imports.

File: flow/util/test/test_genElapsedTime.py [hunk 2]
```diff
@@ -28,23 +30,24 @@ def test_elapsed_time(self, mock_stdout):
             f.write("Some log entry\n")
             f.write("Elapsed time: 01:30:00[h:]min:sec.\n")
         # call the script with the test log file
-        sys.argv = ["util/genElapsedTime.py", "--logDir", str(self.tmp_dir.name)]
+        sys.argv = ["./genElapsedTime.py", "--logDir", str(self.tmp_dir.name)]
         with patch.object(sys, 'argv', sys.argv):
-                import genElapsedTime
+            module = importlib.import_module(self.module_name)
```
Comment (vvbandeira) [inline flow/util/test/test_genElapsedTime.py]: Is this missing the `importlib.reload(module)` like on the other tests?

File: flow/util/test/test_genElapsedTime.py
Comment (vvbandeira) [inline flow/util/test/test_genElapsedTime.py:1]: I think this file should be moved to `flow/test`.

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: flow/util/test/test_genElapsedTime.py (hunk 2 above)
Comment (habibayassin) [reply flow/util/test/test_genElapsedTime.py]: I need to reload for the rest of the test cases but the first test case I only need importing

**Comment by vvbandeira** [comment]
@habibayassin I think you missed the DCO check

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**